src/Storages/Kafka/KafkaConsumer.cpp

```diff
@@ -527,4 +542,64 @@ void KafkaConsumer::storeLastReadMessageOffset()
     }
 }
 
+void KafkaConsumer::setExceptionInfo(const cppkafka::Error & err)
+{
+    setExceptionInfo(err.to_string());
+}
+
+void KafkaConsumer::setExceptionInfo(const String & text)
+{
+    std::lock_guard<std::mutex> lock(exception_mutex);
+    exceptions_buffer.push_back({text, static_cast<UInt64>(Poco::Timestamp().epochTime())});
+}
+
+/*
+ * Needed until
+ * https://github.com/mfontanini/cppkafka/pull/309
```

Comment:
Looks like this got merged https://github.com/mfontanini/cppkafka/pull/309
And it worth to update submodule and remove this function.

Reply:
In process.